tests/NuGet.Services.AzureSearch.FunctionalTests/Support/NuGetSearchFunctionalTestBase.cs

```diff
@@ -63,6 +65,19 @@ protected async Task<IReadOnlyList<string>> SearchAsync(string query, bool inclu
             }
         }
 
+        protected async Task<V2SearchResult> V2SearchAsync(V2SearchBuilder searchBuilder)
+        {
+            using (var response = await Client.GetAsync(searchBuilder.RequestUri))
+            {
+                response.EnsureSuccessStatusCode();
+
+                var json = await response.Content.ReadAsStringAsync();
```

Comment:
I think `ITestOutputHelper` should be passed in here to log the URL and maybe even the response body

Reply:
I do something like this for E2E
https://github.com/NuGet/NuGet.Services.EndToEnd/blob/f6895c206c14a5092998cdca5b932fa46c73f19c/src/NuGet.Services.EndToEnd/Support/Clients/SimpleHttpClient.cs#L46-L66